### DISCUSSION TITLE
Does docsify support specifying a tag or branch to use for generating the website?

### DISCUSSION THREAD

**Discussion by olalonde** [docsifyjs/docsify#1999] [Q&A]
It wasn't clear from the docs whether that was supported. It would be useful for projects that use master branch as the development branch but want to offer documentation for the last release instead (e.g. using git tag `latest`).

**Comment by Koooooo-7** [comment]
Hi @olalonde , If you use the Github Pages or other services, I think it should be the deploy approach of the deploy kit since it only depends on which version docsify docs you wanna use to setup the site. Maybe you could found the trick on github pages or other services related docs.

**Comment by olalonde** [comment]
How would I do that here? Doesn't `docsifyjs/docsify` always refer to the main branch?

```
 window.$docsify = {
    repo: 'docsifyjs/docsify',
    maxLevel: 3,
    coverpage: true,
  };
```

**Reply by Koooooo-7** [reply]
the config of [`repo`](https://docsify.js.org/#/configuration?id=repo)  just determines where to go for the github corner icon of docsify. it does not decide the deploy branch/version. if you deploy your site by github pages. you should check the config of your github pages and config it deploy from a branch or other versions via `github action`.
